### DISCUSSION TITLE
Welcome to base-stack Discussions!

### DISCUSSION THREAD

**Discussion by AlemTuzlak** [forge-42/base-stack#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cweekly** [comment]
### Introduction
Hi! I'm Chris, I've been doing web-related things for a living since 1998. I live in Duxbury (near Cape Cod, south of Boston, MA USA). After spending the last 3 years in a FT role at a big company in a highly-regulated industry, I'm finally returning to running my tiny S-Corp, w/ the freedom to engage w/ the OSS world and indie projects. The forge-42 values and mission really resonate w/ me, and the base-stack is comprised of virtually all the same core libraries I'd likely pick if starting from scratch. I was an early Remix supporter and have always appreciated its "the web is the platform" approach. Thank you, maintainers, for publishing your stuff!

### Question
EDIT: I just moved my question from this comment to a new discussion: [https://github.com/forge-42/base-stack/discussions/54](https://github.com/forge-42/base-stack/discussions/54)
